component/pom.xml

```diff
@@ -519,14 +520,16 @@
                         <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
                         <Bundle-Name>${project.artifactId}</Bundle-Name>
                         <Export-Package>
-                            io.siddhi.extension.io.cdc.*;version="${project.version}"
+                            io.siddhi.extension.io.cdc.*;version="${project.version}",
```

Comment:
Why are you both private packaging and exporting. Isn't these dependencies being used within your extension

Reply:
they are required for a dependency at the moment without this the extension won't work for Oracle CDC

Reply:
The issue was due to kafka connect-runtime.jar having spi services fixed by copying the services into the manifest folder